x-pack/plugins/observability/server/lib/rules/threshold/register_threshold_rule_type.ts

```diff
@@ -180,5 +181,8 @@ export function thresholdRuleType(
     },
     producer: observabilityFeatureId,
     alerts: MetricsRulesTypeAlertDefinition,
+    getViewInAppRelativeUrl: ({ rule }: GetViewInAppRelativeUrlFnOpts<{}>) => {
```

Comment:
@mikecote Not sure if we can do this typing in a more elegant way. Please let me know if you have another idea rather than exposing `GetViewInAppRelativeUrlFnOpts` type from the alerting plugin.

Reply:
👍 sorry for the delay, as a follow up from our offline discussion, an issue will be created to see how the framework can simplify this.